### PR TITLE
doc: Cleanup serialization.md, #24717

### PR DESCRIPTION
**Pull request by patriknw**
Refs #24717

### CODE REVIEW

File: akka-docs/src/main/paradox/typed/cluster.md [hunk 1]
```diff
@@ -291,27 +312,6 @@ be necessary to set the node’s status to `Down` in order to complete the remov
 see [Split Brain Resolver](http://developer.lightbend.com/docs/akka-commercial-addons/current/split-brain-resolver.html),
 part of the [Lightbend Reactive Platform](http://www.lightbend.com/platform).
 
-## Serialization
```
Comment (patriknw) [inline akka-docs/src/main/paradox/typed/cluster.md]: moved to serialization.md

File: akka-docs/src/main/paradox/serialization-classic.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+To use Serialization, you must add the following dependency in your project:
+
+@@dependency[sbt,Maven,Gradle] {
+  group="com.typesafe.akka"
+  artifact="akka-actor_$scala.binary_version$"
+  version="$akka.version$"
+}
+
+## Serializing ActorRefs
```
Comment (patriknw) [inline akka-docs/src/main/paradox/serialization-classic.md]: this section was from original serialization.md, moved here without any changes apart from initial sentence.

**Review by johanandren — APPROVED** [review]
LGTM

**Review by chbatey — COMMENTED** [review]
LGTM

**Review by chbatey — APPROVED** [review]

**Pull request merged**